public/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "short_name": "Quid Refraction",
+  "name": "Quid Refraction",
+  "icons": [
+    {
+      "src": "./icons/pwa-192x192.png",
+      "sizes": "192x192",
+      "type": "image/png"
+    },
+    {
+      "src": "./icons/pwa-512x512.png",
+      "sizes": "512x512",
+      "type": "image/png"
+    }
+  ],
+  "start_url": "./index.html",
```

Comment:
This actually should be `/` (the place from where you copied this is wrong too, I need to fix it)

Reply:
Go ahead!